app/src/components/v-checkbox.vue

```diff
@@ -23,6 +23,8 @@ interface Props {
 	block?: boolean;
 	/** If a custom value can be entered next to it */
 	customValue?: boolean;
+	/** Will focus the custom value input on mounted */
+	focusCustomInput?: boolean;
```

Comment:
Lets name this `autofocus` to match [the html standard](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/autofocus)